linetools/analysis/tests/test_continuum.py

```diff
@@ -5,10 +5,11 @@
 from ..continuum import find_continuum
 import imp
 import numpy as np
+import pytest
```

Comment:
Not really, but it doesn't hurt and 
pytest.set_trace() is the same as pdb.set_trace() 
so it replaces an (unnecessary) import pdb.